.github/ISSUE_TEMPLATE/maintainerRequest.yml

```diff
@@ -0,0 +1,83 @@
+name: Maintainer Request
+description: Request membership to help maintain this repository.
+title: Maintainer Request
+labels: ['maintainer-request']
+projects: ['repo-management']
```

Comment:
I think this should add to a project across all of the DSCCommunity & also one that is just in the AzureDevOps repo 

This gives repo level and org level visibility very easily without needing to use additional automation or manual tasks to do so